### ISSUE TITLE
[v0.1alpha] TreePanel: Crash on reverse lookup

### ISSUE THREAD

**Issue by SillyBits** [SillyBits/satisfactory-savegame-tool-ng#2]
From error report # 28

```
Message: Der Wert darf nicht NULL sein.
Parametername: key
TargetSite: ThrowArgumentNullException, C:\Windows\Microsoft.NET\Framework64\v4.0.30319\mscorlib.dll
Source: mscorlib
StackTrace:
   bei System.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
   bei System.Collections.Generic.Dictionary`2.FindEntry(TKey key)
   bei SatisfactorySavegameTool.Panels.BasicTree._Details_Modified(Property prop)
   bei SatisfactorySavegameTool.Panels.DetailsPanel.ModifiedHandler.Invoke(Property prop)
   bei SatisfactorySavegameTool.Panels.Details.FloatControl.OnPropertyChanged(DependencyPropertyChangedEventArgs e)
   bei System.Windows.DependencyObject.NotifyPropertyChange(DependencyPropertyChangedEventArgs args)
```

**Comment by SillyBits** [comment]
Might have been raised from one of those specialized viewer, most child properties aren't registered within the tree(s)

**Comment by SillyBits** [comment]
Fixed with 1e5de2e
But more of a workaround until specialized viewers are capable of reporting modifications correctly
